internal/installcfg/installcfg.go

```diff
@@ -132,6 +136,10 @@ type InstallerConfigBaremetal struct {
 	SSHKey                string               `yaml:"sshKey"`
 	AdditionalTrustBundle string               `yaml:"additionalTrustBundle,omitempty"`
 	ImageContentSources   []ImageContentSource `yaml:"imageContentSources,omitempty"`
+	Capabilities          struct {
```

Comment:
I prefer to avoid the anonymous struct here, similar to how the installer does it

Reply:
Sure, changed it